mmpose/models/utils/realnvp.py

```diff
@@ -0,0 +1,50 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import torch
+import torch.nn as nn
+
+
+class RealNVP(nn.Module):
```

Comment:
Is the implementation of RealNVP adapted from other projects? If so, we may need to add license and attribution information properly.